2-mandatory/1-writers.js

```diff
@@ -71,7 +77,8 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+    let result = writers.filter(element => !element.alive && element.age >= 40 && element.age <= 49)[0];
```

Comment:
Very advance. I am impressed :) well done.

Your solution is good! However, one caveat about this line of code is that, what happens if there are no writers which satisfy such condition? The array returned by the filter method will be empty, so calling [0] will not work. Therefore, in the real web development world (or programming world), we always check if the array contains the minumum number of array before calling it to prevent errors. Hope that makes sense!